setup.cfg

```diff
@@ -37,8 +37,7 @@ python_requires = >=3.7
 install_requires =
     django >= 3.2
     django-privates >= 1.5
-    pyopenssl >= 22
-    cryptography >= 35
+    cryptography >= 35.0.0
```

Comment:
we no longer *require* the latest version, but it's still wise to keep dependencies updated. I've reverted the version bump.